README.md

```diff
@@ -1,5 +1,5 @@
 # Pull Request Tutorial
 
 For educational purposes, this is a chunk of text that needs to be formatted
-just a little better (and have its typos fixed).
+just a little bettjklkjljkjkljkljlkjkled).
```

Comment:
This is some serious volapyke... please use better words :)

Reply:
Ooops my bad.